precode.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p /task/dir1 /task/dir2 /task/dir3/dir4
```

Comment:
Не нужно использовать корень файловой системы
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```